frontend/ui/widget/container/inputcontainer.lua

```diff
@@ -269,6 +270,58 @@ function InputContainer:onGesture(ev)
     end
 end
 
+-- Filtered variant that only lets is_always_active touch zones through.
+-- (This is used by the "toggle_touch_input" Dispatcher action).
+function InputContainer:_onGestureFiltered(ev)
+    for _, tzone in ipairs(self._ordered_touch_zones) do
+        if PluginShare.isGestureAlwaysActive(tzone.def.id) and tzone.gs_range:match(ev) and tzone.handler(ev) then
+            return true
```

Comment:
Feels a bit awkward to have InputContainer dependant on this remote pluginshare module. (My preferred way for passing stuff around is to just store the thing in the ubiquitous UIManager without anybody knowing :)
(Also here, `PluginShare.isGestureAlwaysActive()` may crash if the gestures plugin is disabled and it is not set?)
Couldn't the gestures plugin just plug things in InputContainer ?

Reply:
That looks sensible, yeah ;).

Reply:
A fun quirk of the dummy approach here is that it probably shouldn't be a plain NOP, it might be safer to make it return true?